### PR TITLE
[MINOR][DOC] Fix html tag broken in configuration.md

### PR DESCRIPTION
**Pull request by HeartSaVioR**
## What changes were proposed in this pull request?

This patch fixes wrong HTML tag in configuration.md which breaks the table tag.

This is originally reported in dev mailing list: https://lists.apache.org/thread.html/744bdc83b3935776c8d91bf48fdf80d9a3fed3858391e60e343206f9@%3Cdev.spark.apache.org%3E

## How was this patch tested?

This change is one-liner and pretty obvious so I guess we may be able to skip testing.

### CODE REVIEW

**Comment by HeartSaVioR** [comment]
I guess we need to port back to `branch-2.4` and re-render 2.4.1 docs.

**Comment by cloud-fan** [comment]
LGTM, cc @dbtsai

**Review by srowen — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Merged to `master/branch-2.4/branch-2.3`.

**Comment by dongjoon-hyun** [comment]
cc @maropu , too. We need to fix 2.3.3 doc, too.
- https://spark.apache.org/docs/2.3.3/configuration.html#available-properties

**Comment by dongjoon-hyun** [comment]
I'll make a PR on spark-website.

**Comment by dongjoon-hyun** [comment]
The PR is ready.
- https://github.com/apache/spark-website/pull/192

**Comment by dbtsai** [comment]
Thanks all!

**Comment by felixcheung** [comment]
@dongjoon-hyun could you merge this to branch-2.2 also, to match spark-website?

**Comment by dongjoon-hyun** [comment]
Nope. @felixcheung . `branch-2.2` is EOL. We should give a clear message to the Apache Spark community by not backporting even in the case of bugs.

**Comment by srowen** [comment]
It wouldn't be crazy to patch up an error in the 2.2 docs, but yeah I doubt it matters much now that it's eol.

**Comment by felixcheung** [comment]
Sure, just FYI it’s not really Kosher to patch up the doc release without having the change committed to the Apache git repo. Even if end of life. My vote would have been to leave 2.2 doc as is, even if it’s broken.

**Comment by srowen** [comment]
I agree with that general principle but this was a clear fix to how the released docs are intended to render. spark-website's copy isn't the official copy though the baseline expectation is indeed that it hosts just what is in the release. To the extent these changes need committer or PMC approval, they have it.

**Comment by dongjoon-hyun** [comment]
I agree with @srowen . But, @felixcheung , you can make a PR on `spark-website` by yourself. If there is your PR, I'll gave +1.

**Pull request closed without merging**
